Makefile

```
@@ -67,8 +67,14 @@ flamegraph:
 compare_benchmarks: $(COMPILED_BENCHES)
 	cd bench && ./run_benchmarks.sh
  
-compare_vm_output: $(CLEO_TRACE) $(CAIRO_TRACE) $(CLEO_MEM) $(CAIRO_MEM)
-	cd tests; ./compare_vm_output.sh
+compare_trace_memory: $(CLEO_TRACE) $(CAIRO_TRACE) $(CLEO_MEM) $(CAIRO_MEM)
+	cd tests; ./compare_vm_state.sh trace memory
+
+compare-trace: $(CLEO_TRACE) $(CAIRO_TRACE)
+	cd tests; ./compare_vm_state.sh trace
+
+compare-memory: $(CLEO_MEM) $(CAIRO_MEM)
+	cd tests; ./compare_vm_state.sh memory
```

Let's be consistent about target naming:
```suggestion
compare_trace: $(CLEO_TRACE) $(CAIRO_TRACE)
	cd tests; ./compare_vm_state.sh trace

compare_memory: $(CLEO_MEM) $(CAIRO_MEM)
	cd tests; ./compare_vm_state.sh memory
```